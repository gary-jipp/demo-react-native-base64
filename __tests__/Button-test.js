import 'react-native';
import React from 'react';
import Button from 'Button';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('Button Tests', () => {

  it('Renders and Matches snapshot', () => {
    const tree = renderer.create(<Button title="Press Me" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

});
