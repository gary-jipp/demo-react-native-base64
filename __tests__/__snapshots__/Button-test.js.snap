// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Tests Renders and Matches snapshot 1`] = `
<View
  accessibilityRole="button"
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
  testID="button"
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#2196F3",
          "borderRadius": 5,
          "elevation": 0,
          "justifyContent": "center",
          "paddingHorizontal": 20,
          "paddingVertical": 12,
        },
        undefined,
        undefined,
      ]
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#EEE",
            "fontSize": 16,
            "fontWeight": "bold",
            "letterSpacing": 0.25,
            "lineHeight": 21,
          },
          undefined,
        ]
      }
    >
      Press Me
    </Text>
  </View>
</View>
`;
